roles/setup_pemagent/tasks/pem_agent_config.yml

```diff
@@ -12,9 +12,9 @@
     PEM_SERVER_PASSWORD={{ pg_pem_admin_password }} \
     {{ pem_agent_bin_path }}/pemworker \
          --register-agent \
-         --display-name {{ item.key }} \
+         --display-name {{ inventory_hostname }} \
          --pem-user {{ pg_pem_admin_user }} \
-         --pem-server {{ pem_server_private_ip }} \
+         --pem-server {{ hv.pem_server_private_ip }} \
```

Comment:
Same as mentioned about the short name versus meaningful name.

Reply:
Indeed. bc36959 fixes this